config/default/kustomization.yaml

```diff
@@ -1,12 +1,12 @@
 # Adds namespace to all resources.
-namespace: maistraoperator-system
+namespace: istio-operator
 
 # Value of this field is prepended to the
 # names of all resources, e.g. a deployment named
 # "wordpress" becomes "alices-wordpress".
 # Note that it should also match with the prefix (text before '-') of the namespace
 # field above.
-namePrefix: maistraoperator-
+# namePrefix: istiooperator-
```

Comment:
Intended change?

Reply:
yeah, I needed to get rid of the prefix

Reply:
ok, let me put in a different way: Should we remove this field entirely (including the comments above it)? Or just commenting it is enough meaning this is a temporary change?

Reply:
yeah it's a good question. I decided to leave it in while we're figuring things out. We'll need a big cleanup anyway, there's a lot of commented out stuff, and a lot of boilerplate we don't need. I'll create a ticket to cover that